ds_judgements_public_ui/static/js/src/modules/date_input.js

```diff
@@ -0,0 +1,26 @@
+import $ from "jquery";
+
+function constrainInputToMinMax() {
```

Comment:
It would be worth testing the event listener. Do we have any tests for other event listeners in the repo?

Reply:
aha, we have no js tests at all currently (and very little js). I agree that would be useful though - do you favour any particular JS testing framework / tool?

Reply:
no preference, but Im familiar with jest. I've most recently been doing react where we used react-testing-library, but in this scenario I think we can use DOM Testing Library.

But again, no preference

Reply:
ok, good stuff - I'll take a look into Jest! do you mind if i add a ticket to improve JS test coverage and punt this conversation over there?

Reply:
If we're starting to push functionality into the JS then I'm a big fan of adding something like jest to prove that it's doing what we think.

If we could add this to both EUI and PUI at the same time (even if EUI literally just does a 'hello world' type test) that would be great, so they're using the same framework and config.

Reply:
Yeah sure thing @timcowlishaw, do let me know if you'd like some help with that too